conu/apidefs/metadata.py

```diff
@@ -8,6 +8,15 @@ class Metadata(object):
 
     def __init__(self, name=None, identifier=None, labels=None, command=None,
                  creation_timestamp=None, env_variables=None):
+        """
+        :param name: str, name of object
+        :param identifier: int, id of object
+        :param labels: dict, {key: value}
+        :param command: str, command to be executed
```

Comment:
this is worth debating: `str` or a `list` of `str`s?

Reply:
+1 for `list` of `str`s : 
- `["python3", "-m", "http.server", "--bind", "0.0.0.0", "8080"]`

Reply:
yup, we can even easily transform to plain str if some backend uses such syntax; going from str to list of strs is much harder